pkg/rtc/transportmanager.go

```diff
@@ -20,6 +21,12 @@ import (
 
 const (
 	failureCountThreshold = 2
+
+	// when RR report loss frac over this threshold, we consider it is a unstable event
+	udpLossFracUnstable = 25
+	// if in last 32 times RR, the unstable report count over this threshold, the connection is unstable
+	udpLossUnstableCountThreshold = 20
+	tcpGoodRTT                    = 80
```

Comment:
Should we add a check for RTCP RR report with min number of packets. With DTX, there might be windows with small number of packets.

Reply:
Don't want to count packet numbers in transport manager, might need do it in other place(downtrack or medialossproxy), will draft another PR to refine that.